src/opm/output/eclipse/AggregateUDQData.cpp

```diff
@@ -192,6 +585,7 @@ namespace {
         {
             int l_sstr = 8;
             int max_l_str = 128;
+            std::string temp_str = "";
```

Comment:
The `std::string` is default initialized to empty string - can just do `std::string temp_str;`

Reply:
Agree - done!